Readme.md

```diff
@@ -31,6 +31,75 @@ Place your template files (.cshtml or .liquid) to the *Views/VueComponents* fold
+- The script must export the module as an object literal ESM style (`export default { ... }`).
+  - You don't need to specify the `name` and `template` properties, as these are automatically provided during compilation.
+- If your component has child components, include the _.vue_ extension when importing them. 
+
+For example if you have the file _My.Module/Assets/Scripts/VueComponents/my-article.vue_:
+```vue
+<template>
+    <article>
+        <header>
+            <h3>{{ title }}</h3>
+        </header>
+        <slot></slot>
+        <footer>{{ formattedDate }}</footer>
+    </article>
+</template>
+
+<script>
+export default {
+    props: [ 'title', 'date' ],
+    computed: {
+        formattedDate(self) {
+            const formatter = new Intl.DateTimeFormat(
+                self.culture,
+                {
+                    year: 'numeric',
+                    month: 'short',
+                    day: 'numeric',
+                });
+
+            return formatter.format(self.date);
+        }
+    }
+};
+</script>
+```
+
+You can include the `<vue-component name="my-article">` tag helper in your code. This will add Vue and _My.Module/wwwroot/vue/my-article.js_ to the resource manager (using the `vue-component-{name}` resource) as well as the `VueComponent-MyArticle` shape (the SFC's kebab-case name is converted into PascalCase). Include `depends-on="vue-component-my-article"` in your app's `<script>` element to ensure correct load order.
+The Rollup plugin automatically registers each component you include (but not their children) as globally accessible components. So you don't need to list them in your app's `components` property. Indeed the component object isn't exposed as a global variable.
+
+
+### Advantages of SFCs.
+
+- Tooling! If you have an IDE plugin for Vue.js it will work better. Syntax highlighting, property autocomplete, Go to Definition for custom elements, and all other advantages of static Vue development.
+- The script and template are kept together, which makes understanding the individual component easier.
+- No need to escape the `@` on events.
+
+
+### Limitations and Considerations
+
+- No other Razor features including string localizer with arguments.
+- Including a script element in your template will break it. Although you shouldn't do that anyway.
+- As you might expect from Orchard Core, the style element isn't supported either since you will be using themes. If you can think of a use-case that's applicable for OC, please open an issue. 
+
+Regarding the points: if you need anything more complicated, first reconsider you application design to see if your goals can be achieved in a more Vue.js logic. For example pass the variables in your main app that hands them down via property binding. If you still need something else, either use a _cshtml_ templated app as outlined above or use shape overriding on the `VueComponent-{FileNameInPascalCase}` shape.

```

Comment:
So am I correct that the shape overriding is only possible by overriding this shape in a .cshtml file? So it's not possible to override a .vue file, correct?